### PR TITLE
Keyman Desktop and Keyman Developer open source 10.0 alpha seed

### PR DESCRIPTION
**Pull request by mcdurdin**

### CODE REVIEW

File: windows/src/buildtools/help/main/upload.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
+
+<section id="upload">
+	<title>Upload Process</title>
+
+	<para>The following steps are required for uploading the new version to the server:</para>
+
+	<orderedlist>
+		<listitem>Upload programs: Upload keymandeveloper7-0-x-0.exe, keymandesktoplight7-0-x-0.exe, keymandesktoppro7-0-x-0.exe to s:\tavultesoft\website\dec2004\downloads</listitem>
```
Comment (darcywong00) [inline windows/src/buildtools/help/main/upload.xml]: Are these items still relevant to the open-source repo?

Comment (mcdurdin) [inline windows/src/buildtools/help/main/upload.xml]: Nope ... removed :)

Comment (darcywong00) [inline windows/src/buildtools/help/main/upload.xml]: 👍
